src/Simulation/Native/src/simulator/wavefunction.hpp

```diff
@@ -274,7 +362,8 @@ class Wavefunction
     template <class A>
     bool subsytemwavefunction(std::vector<unsigned> const& qs, std::vector<T, A>& qubitswfn, double tolerance)
     {
```

Comment:
Why have you extracted flush from Fused::subsytemwavefunction? Is keeping that function still useful?

Reply:
Removed it to streamline flush() call through the repo

Reply:
Is keeping Fused::subsytemwavefunction still useful?